### DISCUSSION TITLE
Automatically create a graph of links/backlinks?

### DISCUSSION THREAD

**Discussion by Dob4115** [RobertWeinmeister/dokuwiki-mermaid#24] [Ideas]
This seems like a cool plugin. Is there any intent to make it grab a specified depth from the current page, using links/backlinks? I could see this being put in the page itself, or in the sidebar.

**Comment by RobertWeinmeister** [comment]
Could you describe this in more detail? I am not sure what it is exactly that you have in mind. Maybe give an example.

**Reply by Dob4115** [reply]
So the general idea is that wikis are graphs but few (if any?) have a means of visualizing the graph builtin. I think Obsidian does... but the idea is that, "realtime" you could see how the current pages, namespace, etc. relate to other pages (a picture of the graph of pages related to the current page via links/backlinks). Of course, I imagine the graph picture creation is CPU intensive. So being able to set a limit on how many levels away from the current page are displayed.

**Reply by RobertWeinmeister** [reply]
I understand your idea now. This could certainly be done with mermaid and therefore this plugin, but this plugin only provides the means to do that. Any actual implementation is not the scope of this plugin.
